__docs__/wonder-blocks-dropdown/single-select.argtypes.ts

```diff
@@ -16,15 +28,15 @@ const argTypes: ArgTypes = {
     labels: {
         control: {type: "object"},
         description:
-            "The object containing the custom labels used inside this component.",
+            "The object containing the custom values used inside this component.",
```

Comment:
```suggestion
            "The object containing the custom label values used inside this component.",
```
Minor suggestion to make it more specific to avoid confusion since the `OptionItem` component has a `value` prop! (same for MultiSelect!)

Reply:
`label values` works for me! It's also confusing that an `OptionItem`'s content becomes a value in the opener...